dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/dotgenerator/DotCpg14Generator.scala

```diff
@@ -15,7 +15,7 @@ object DotCpg14Generator {
     val cfg = new CfgGenerator().generate(method)
     val ddg = new DdgGenerator().generate(method)
     val cdg = new CdgGenerator().generate(method)
-    DotSerializer.dotGraph(method, ast ++ cfg ++ ddg ++ cdg, withEdgeTypes = true)
+    DotSerializer.dotGraph(Some(method), ast ++ cfg ++ ddg ++ cdg, withEdgeTypes = true)
```

Comment:
```suggestion
    DotSerializer.dotGraph(Option(method), ast ++ cfg ++ ddg ++ cdg, withEdgeTypes = true)
```

Always prefer `Option(xyz)` over `Some(xyz)` because it performs a null-check, therefor you'll never end up with a `Some(null)`. There's more occurrences in this PR, can you please change them as well?